src/main/java/org/jboss/xavier/integrations/rbac/RBACRouteBuilder.java

```diff
@@ -58,7 +58,8 @@ public void configure() throws Exception {
                 .setHeader(RBAC_NEXT_LINK, constant(""))
                 .loopDoWhile(exchange -> exchange.getIn().getHeader(RBAC_NEXT_LINK) != null)
                     .setHeader(Exchange.HTTP_METHOD, constant(HttpMethods.GET))
-                    .setHeader(Exchange.CONTENT_TYPE, constant("application/json"))
+                    .setHeader(Exchange.CONTENT_TYPE, constant("application/json")) // Content-type is a way to specify the media type of request being sent from the client to the server (valid on POST and PUT)
+                    .setHeader("Accept").simple("application/json") // Accept header is a way for a client to specify the media type of the response content it is expecting.
```

Comment:
Same as above with the addition of using `constant` since it's not changing.
```suggestion
                    .setHeader(HttpHeaders.ACCEPT, constant(ContentType.APPLICATION_JSON)) // Accept header is a way for a client to specify the media type of the response content it is expecting.
```